src/groupsplit.py

```diff
@@ -326,14 +345,34 @@ def main():
     logger.setLevel(log_levels[options.verbosity])
     splitwise = Splitwise(options.api_client)
     split_gen = SplitGenerator(options, args, splitwise)
-    print "Uploading splits"
+    print("Uploading splits")
     for uri in split_gen:
         if options.dryrun:
-            print uri
+            print(uri)
             continue
         splitwise.post_expense(uri)
     sys.stdout.write("\n")
     sys.stdout.flush()
 
+def dump_cats():
+    """
+    Get list of category IDs and names for reference
+    """
+    splitwise = Splitwise()
+    cats = splitwise.get_categories()
+    # Extract ID and Name from the list of dictionaries
+    result = []
+    for item in cats:
+        result.append({'id': item['id'], 'name': item['name']})
+        if 'subcategories' in item:
+            for subcategory in item['subcategories']:
+                result.append({'id': subcategory['id'], 'name': subcategory['name']})
+    file_name = '../test/all_categories_list.csv'
```

Comment:
Files downloaded and used during runtime should go into a temporary folder ignored by git. Do categories in SW change per user or are they static? If it's the latter there's no need for this function.

Reply:
I believe it's static so I'll pull this out